src/Symfony/Bundle/SecurityBundle/Tests/Functional/app/JsonLoginLdap/config.yml

```diff
@@ -21,7 +22,7 @@ security:
                 search_password: ''
                 default_roles: ROLE_USER
                 uid_key: uid
-                extra_fields: ['email']
+                #extra_fields: ['email']
```

Comment:
Still needed or should be removed?

Reply:
I am not sure. I will check if still needed.